modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/SampleFDSPropsTransformer.js

```diff
@@ -30,6 +30,44 @@ export default function propsTransformer({
 			if (action.data.id === 'sampleMessage') {
 				alert(`${greeting} ${itemData.title}!`);
 			}
+			else if (action.data.id === 'sampleEditMessage') {
+				openModal({
+					bodyHTML: `
+						<form>
+							<div class="form-group">
+								<label for="title">Title</label>
+								<input
+									class="form-control"
+									id="title"
+									placeholder="Title"
+									type="text"
+									value="${itemData.title}"
+								/>
+							</div>
+							<div class="form-group">
+								<label for="description">Description</label>
+								<input
+									class="form-control"
+									id="description"
+									placeholder="Description"
+									type="text"
+									value="${itemData.description}"
+								/>
+							</div>
+						</form>
+					`,
+					buttons: [
+						{
+							label: 'OK',
+							onClick: ({processClose}) => {
+								processClose();
+							},
+						},
+					],
+					size: 'md',
+					title: 'Sample Edit Modal',
+				});
+			}
```

Comment:
We need a second action to enable the quick actions... If there's only one, it shows directly instead of the ellipsis button. I agree this is maybe too much stuff for an example but I do not want to repeat the alert one.
Any suggesstions?

Reply:
If it's just to add any action item, we don't need the custom handling of the action. We can just add a `FDSActionDropdownItem` that doesn't do anything.

Reply:
Continues on https://github.com/liferay-frontend/liferay-portal/pull/2222